src/Microsoft.AspNet.Mvc.Core/Rendering/HtmlHelperOptions.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Mvc.Core;
+
+namespace Microsoft.AspNet.Mvc.Rendering
+{
+    public class HtmlHelperOptions
+    {
+        private string _idAttributeDotReplacement = "_";
+        private bool _clientValidationEnabled = true;
+        private string _validationMessageElement = "span";
+        private string _validationSummaryMessageElement = "span";
+
+        /// <summary>
+        /// Set this property to <see cref="Html5DateRenderingMode.Rfc3339"/> to have templated helpers such as
```

Comment:
Please word all property summaries in this new class consistently -- "Gets or sets ..."